css/style.css

```diff
@@ -0,0 +1,6 @@
+img{
+    max-width: 80%;
+    height: auto;
+    text-align: center;
+    display: block;
+}
```

Comment:
Great job being polite, however I kindly suggest that you give a reason for the requested change.

### EXAMPLE
Kindly add a new line after closing brockets because it makes your code cleaner.

Reply:
Kindly add linters to your project, it will show you the mistakes in your coding.